### PR TITLE
Fix System log and settings in Admin

### LINKED ISSUE #2210: Fix System log and settings in Admin
**Actual Behaviour**
Currently the system settings table is not visible completely in mobile view. Fix UI of system log in mobile view.

**Expected Behaviour**
Make the table scrollable like other tables.
Improve the UI of system log in mobile view.

**Screenshots of the issue**
Screenshot 2019-06-06 at 9 34 11 AM
Screenshot 2019-06-06 at 9 36 19 AM

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2210  and part of #2219 

Changes: Improved the UI of system settings and logs in mobile view.
Demo Link: https://pr-2236-fossasia-susi-web-chat.surge.sh

Screenshots of the change: [Add screenshots depicting the changes.]
Screenshot 2019-06-09 at 1 26 25 PM
Screenshot 2019-06-09 at 1 26 39 PM

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd Shall I remove the antd tables and implement them the same way as in system settings ? Also instead of having scrolling app bar can we have a dropdown ?

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd Please review. I have fixed the system settings and logs in mobile view. Also the top appbar now has arrows and scrolls in mobile view

File: src/components/Admin/SystemLogs/SystemLogs.js [hunk 1]
```diff
@@ -8,15 +8,31 @@ import urls from '../../../utils/urls';
 import CircularProgress from '@material-ui/core/CircularProgress';
 import Alert from '../../shared/Alert';
 import styled from 'styled-components';
+import isMobileView from '../../../utils/isMobileView';
+
+const mobileView = isMobileView();
```
Comment (fragm3) [inline src/components/Admin/SystemLogs/SystemLogs.js]: MobileView not required here, use media queries

Comment (fragm3) [inline src/components/Admin/SystemLogs/SystemLogs.js]: MobileView not required here, use media queries

Comment (harsh-jindal) [reply src/components/Admin/SystemLogs/SystemLogs.js]: @fragm3 I passing left as a prop to shared Alert component. So I guess I cannot use meadia queries

Comment (harsh-jindal) [reply src/components/Admin/SystemLogs/SystemLogs.js]: @fragm3 The screenshot you have attached will occur if you do not reload in mobile view. There is no UI problem if we reload it in mobile view. Please check

**Comment by harsh-jindal** [comment]
@fragm3 Please review.

File: src/components/Admin/SystemLogs/SystemLogs.js (hunk 1 above)
Comment (fragm3) [reply src/components/Admin/SystemLogs/SystemLogs.js]: I don't understand the point of using props for it and passing it down to alert component. You could have directly used media queries in alert component.

**Comment by harsh-jindal** [comment]
@fragm3 Made the changes.

**Comment by harsh-jindal** [comment]
@akshatnitd Can you please help here ?
Screenshot 2019-06-16 at 8 38 48 AM

**Comment by harsh-jindal** [comment]
@akshatnitd Works perfectly in my local. Will try to debug why it's happening in surge link.

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd Fixed the issue regarding white overlay. Please review

**Pull request merged**